packages/lib/src/components/ApplePay/ApplePay.tsx

```diff
@@ -53,6 +54,11 @@ class ApplePayElement extends UIElement<ApplePayElementProps> {
     }
 
     submit() {
+        // Analytics
+        if (this.props.isInstantPayment) {
+            this.submitAnalytics({ type: ANALYTICS_SELECTED_STR, target: 'instant_payment_button' });
+        }
+
```

Comment:
Is this supposed to be 'selected' or 'clicked'? 
Here we are tracking when the button is clicked.

Reply:
'selected' is correct. That's what's been decided between the SDKs - we want to know when an instant PM has bee n chosen by the shopper